melodies_monet/driver.py

```diff
@@ -474,6 +510,8 @@ def read_control(self, control=None):
         if 'output_dir' in self.control_dict['analysis'].keys():
             self.output_dir = self.control_dict['analysis']['output_dir']
         self.debug = self.control_dict['analysis']['debug']
+        if 'resample' in self.control_dict['analysis'].keys():
```

Comment:
In the latest commit I moved the resampling to the obs class. In the control file, the resample value is now set under the type of obs. This would also allow for setting different resample intervals for different obs types if someone ever had a need for that.